README.md

```diff
@@ -92,3 +92,8 @@ computations.
 [StreamZip]: https://pub.dev/documentation/async/latest/async/StreamZip-class.html
 [SubscriptionStream]: https://pub.dev/documentation/async/latest/async/SubscriptionStream-class.html
 [typedStreamTransformer]: https://pub.dev/documentation/async/latest/async/typedStreamTransformer.html
+
+## Publishing automation
```

Comment:
WDYT about adding a CONTRIBUTING.md file with this information? It isn't very relevant to consumers of this package on pub.

Reply:
Yeah, makes sense. I mostly want to remove any uncertainty about how to contribute to the repo - the form the PR should be in. If we don't want a brief blurb in the readme, a combo of a section in the contributing doc, and having a link written into the publish bot's PR comments should work.

Reply:
I may land this as is and figure out the right balance between the readme and the contributing docs asynchronously.